tools/test.py

```diff
@@ -68,14 +68,17 @@ def parse_args():
         help='evaluation metrics, which depends on the dataset, e.g., "bbox",'
         ' "segm", "proposal" for COCO, and "mAP", "recall" for PASCAL VOC')
     parser.add_argument('--show', action='store_true', help='show results')
+    parser.add_argument(
+        '--out-dir',
```

Comment:
It might be a little bit misleading since the `test.py` already have an option `--out` but `--out_dir` does totally different things. Would `--show_dir` be better?

Reply:
I guess that `show_dir` is a better option. I used images_out_dir befor but maybe it was too long.